app/collector/data_collection_engine.py

```diff
@@ -0,0 +1,193 @@
+
+import logging
+from datetime import datetime
+import json
+from app.collector.utils.request_utils import (
+    post_request,
+    patch_request,
+    get_request
+)
+from app.collector.utils.cron_utils import (
+    check_if_cron_is_today
+)
+
+
+import app.collector.utils.constants_utils as constants
+from app.collector.collector import (Collector, GenericTable)
+from app.collector.collector_factory import CollectorFactory
+from app.collector.data_collection_logging import DataCollectionLogging
+from app.collector.data_collection_diff_checker import DataCollectionDiffChecker
+
+import os
+
+class DataCollectionEngine():
+# Class to implement the collection data engine.
+
+    def __init__(self):
+        self.log = DataCollectionLogging()
+        self.diff = DataCollectionDiffChecker(self.log)
+
+    def _format_fully_qualified_name (self, list_values):
+    # Format the fully qualified name.
+        fully_qualified_name = list_values
+        fully_qualified_name = [f.lower().replace(" ", "_") for f in fully_qualified_name]
+        return "-".join(fully_qualified_name)+"99"
```

Comment:
Why +"99"? You're using '-' as separator for fqn? I was using dot ('.'), I think it is more common.